src/Compilers/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -3577,9 +3577,9 @@ static Microsoft.CodeAnalysis.ModelExtensions.GetTypeInfo(this Microsoft.CodeAna
 static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromFile(string! path) -> Microsoft.CodeAnalysis.ModuleMetadata!
 static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromImage(System.Collections.Generic.IEnumerable<byte>! peImage) -> Microsoft.CodeAnalysis.ModuleMetadata!
 static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromImage(System.Collections.Immutable.ImmutableArray<byte> peImage) -> Microsoft.CodeAnalysis.ModuleMetadata!
-static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromImage(System.IntPtr peImage, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
-static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(System.IntPtr metadata, int size, System.Action! onDispose) -> Microsoft.CodeAnalysis.ModuleMetadata!
-static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(System.IntPtr metadata, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
+static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromImage(nint peImage, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
+static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(nint metadata, int size, System.Action! onDispose) -> Microsoft.CodeAnalysis.ModuleMetadata!
+static Microsoft.CodeAnalysis.ModuleMetadata.CreateFromMetadata(nint metadata, int size) -> Microsoft.CodeAnalysis.ModuleMetadata!
```

Comment:
Does this mean we can re-enable analyzers here?

https://github.com/dotnet/roslyn/blob/3b23723934d3c7d0b3c1a0165def4a7f0e6e3a19/eng/targets/Settings.props#L152-L153

(Disabled in https://github.com/dotnet/roslyn/pull/67333#discussion_r1151928015.)

Reply:
Let me try that localy.

Reply:
yep can remove that now.